addons/website_sale/data/mail_template_data.xml

```diff
@@ -37,13 +37,6 @@
                         </t>
                         <hr/>
                     </t>
-                    <div style="text-align: center; padding: 16px 0px 16px 0px; font-size: 14px;">
-                        <a t-attf-href="{{ object.get_base_url() }}/shop/cart?access_token={{ object.access_token }}"
-                            target="_blank"
-                            style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
-                            Resume order
-                        </a>
-                    </div>
```

Comment:
I don't understand why this is removed along with all its matching tests and not replaced at all. Not explained at all in the commit message.